package.json

```diff
@@ -99,6 +99,7 @@
     "postcss-loader": "^0.8.0",
     "react": "^0.14.3",
     "react-addons-test-utils": "^0.14.3",
+    "react-dom": "^0.14.3",
```

Comment:
This part is fine. I'd already made the same change in the [v5 branch](https://github.com/bvaughn/react-virtualized/blob/v5/package.json) but it hadn't made its way into master.